src/components/User/Profile/ProfileToggle/MyProfile/MyProfile.js

```diff
@@ -39,11 +39,17 @@ export default class MyProfile extends Component {
         }
 
         const updatedProfileObj = this.handleProfileUpdate(profile, builderObj)
-        
+        console.log("12312312", updatedProfileObj)
         axios.put(`/api/people/${this.props.profile.id}`, {user: updatedProfileObj}).then(res => {
-            const id = this.props.profile.id.toString()
-            this.props.fetchProfileData(id)
+            const userObj = res.data
+            this.props.updateUser(userObj)
+            this.setState({
+                editClicked: !this.state.editClicked
+            })
+        }).catch(err => {
+            console.log("AAAAAAAaa", err)
```

Comment:
If you want to leave this console log please make a better message than 'Aaaaaaa'